schema.sql

```diff
@@ -1,5 +1,3 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE animals( id int GENERATED BY DEFAULT AS Identity, name char(50), date_of_birth date, escape_attempts int, neutered boolean, weight_kg decimal );
```

Comment:
- [OPTIONAL] Great work creating the `animals` table. It would be nice to mark the `id` column as the **_primary key_** column. Using the **_primary key_** would ensure that each record entered into the `animals` table is distinctive 👍 💻